### ISSUE TITLE
Mic and Phone Calls do not work - IOS 14.3

### ISSUE THREAD

**Issue by mastermind9182** [LNSSPsd/PodsGrant#51]
I am on ios 14.3 with unc0ver. I recently just got Airpod Pros 2nd Gen. The mic, receiving calls, and call audio does not work. It just reroutes the audio to my phone. Any solutions to this? I tried to look at the other posts but didn't know if the issue persists between ios versions.

**Comment by LNSSPsd** [comment]
This has been an issue appears randomly across versions that I got no idea with, as the mic works for all of my devices (`iOS 13.3`, `iOS 14.3` and `iOS 15.3.1`) well so that I could not figure out what's wrong. Maybe trying re-pair will help? (probably would not help tho)
I am going to add something like `bluetoothd log exporting` and try to have the logs of the devices whose mic doesn't work to figure out what's wrong with them, but tbh that would be a tough job.

**Comment by mastermind9182** [comment]
Interesting. Well thank you for replying, I do hope that in the future you or somebody else may be able to find a solution! I just bought them today (have been using the Airpods 1, First Gen, so first airpods ever made since they came out) and well if I can't find a solution soon then I may just return them as I do take lots of calls and it would definitely be a hindrance.

**Comment by joseph3896** [comment]
> This has been an issue appears randomly across versions that I got no idea with, as the mic works for all of my devices (`iOS 13.3`, `iOS 14.3` and `iOS 15.3.1`) well so that I could not figure out what's wrong. Maybe trying re-pair will help? (probably would not help tho)
> I am going to add something like `bluetoothd log exporting` and try to have the logs of the devices whose mic doesn't work to figure out what's wrong with them, but tbh that would be a tough job.



> This has been an issue appears randomly across versions that I got no idea with, as the mic works for all of my devices (`iOS 13.3`, `iOS 14.3` and `iOS 15.3.1`) well so that I could not figure out what's wrong. Maybe trying re-pair will help? (probably would not help tho)
> I am going to add something like `bluetoothd log exporting` and try to have the logs of the devices whose mic doesn't work to figure out what's wrong with them, but tbh that would be a tough job.


can you also put it on 14.2.1 ? 
If possible. Many thanks!

**Comment by fazeshow** [comment]
Hi! I have the same issues with my AirPods Pro 2 generation in my iPhone 12 Pro Max on iOS 14.4.1 with uncover jailbreak, @mastermind9182 are you continue with that problem. I connect my AirPods with others device like iphone SE first generation work fine in iOS 15.7 also try in my MacBook and work too! So, I thing that issues is in this version iOS 14. 😅

**Comment by mastermind9182** [comment]
> Hi! I have the same issues with my AirPods Pro 2 generation in my iPhone 12 Pro Max on iOS 14.4.1 with uncover jailbreak, @mastermind9182 are you continue with that problem. I connect my AirPods with others device like iphone SE first generation work fine in iOS 15.7 also try in my MacBook and work too! So, I thing that issues is in this version iOS 14. 😅

Yes the issue is with ios 14 as the airpod pro 2 need a higher version of ios to fully work. We will just have to wait for a fix for it soon. #

**Comment by ChipSkylark37** [comment]
Same issue with iOS 15.0.1. We discussed in another thread and looked through Bluetooth logs but could not determine the cause or how to resolve unfortunately.

**Comment by DmitriiPl** [comment]
I am on ios 14.2.2 with unc0ver
I recorded all the logs in detail for a total of 27 files. My actions: 
1. take the left earphone out of the case and insert it into my ear, animation and connection occur. 
2. make a call, while automatically switching the sound to the speakers.
3. forcibly switch to airdops pro (of course there is no sound)
4. ending the call
5. put the earphone back in the case
[Bluetooth.zip](https://github.com/LNSSPsd/PodsGrant/files/12186791/Bluetooth.zip)

**Comment by plia7** [comment]
> Hi! I have the same issues with my AirPods Pro 2 generation in my iPhone 12 Pro Max on iOS 14.4.1 with uncover jailbreak, @mastermind9182 are you continue with that problem. I connect my AirPods with others device like iphone SE first generation work fine in iOS 15.7 also try in my MacBook and work too! So, I thing that issues is in this version iOS 14. 😅

Hello, I have the same phone ios. Fogu 14 unc0over untether for ios 14.4.1. Is it still an issue for you? Should I not bother getting air pods 2 for my phone? What's the alternative for our phone, Air Pods 1?

**Comment by DmitriiPl** [comment]
Hi. I tried AirPods 3. They work great when calling, there is sound




пятница, 28 июля 2023 г., 02:24 +0300 от ***@***.***  ***@***.***>:

**Comment by plia7** [comment]
> Hi. I tried AirPods 3. They work great when calling, there is sound

What phone and ios do you have? do you have an amazon links to airpods 3? is it regular or pro?

**Comment by DmitriiPl** [comment]
12 pro max ios 14.2.2 fugu. Airpods 3 there is no pro version

**Comment by plia7** [comment]
> 12 pro max ios 14.2.2 fugu. Airpods 3 there is no pro version

will it work for my 12 pro max ios 14.4.1?

**Comment by plia7** [comment]
> 12 pro max ios 14.2.2 fugu. Airpods 3 there is no pro version

can you control the volume from the airpods themselves or you need to use the phone?

**Comment by DmitriiPl** [comment]
I don't know but they will probably work

**Comment by plia7** [comment]
> I don't know but they will probably work
> 
> there is no volume control on airpods 3, but you can use a tweak Siliquo

does Siliquo 2 works in ios 14?

**Comment by DmitriiPl** [comment]
> > I don't know but they will probably work
> > there is no volume control on airpods 3, but you can use a tweak Siliquo
> 
> does Siliquo 2 works in ios 14?

Yes

**Comment by plia7** [comment]
> > > I don't know but they will probably work
> 
> > > there is no volume control on airpods 3, but you can use a tweak Siliquo
> 
> > 
> 
> > does Siliquo 2 works in ios 14?
> 
> 
> 
> Yes. Or Siliquo Pro but something worked

Thank you, do you think buying from amazon is legit?

Apple AirPods (3rd Generation) Wireless Earbuds with Lightning Charging Case. Spatial Audio, Sweat and Water Resistant, Up to 30 Hours of Battery Life. Bluetooth Headphones for iPhone https://a.co/d/5Z2jMNr

**Comment by plia7** [comment]
> > > I don't know but they will probably work
> > > there is no volume control on airpods 3, but you can use a tweak Siliquo
> > 
> > 
> > does Siliquo 2 works in ios 14?
> 
> Yes. Or Siliquo Pro but something worked

I got air pods gen 3, and installed PodsGrant latest version 0.3.0
how can I tell that: 
1. volume slider is working?
2. audio spatial is working?
3. noise cancellation is working?

**Comment by plia7** [comment]
> > > I don't know but they will probably work
> > > there is no volume control on airpods 3, but you can use a tweak Siliquo
> > 
> > 
> > does Siliquo 2 works in ios 14?
> 
> Yes. Or Siliquo Pro but something worked

Can you be more specific which one works for you Silquo 2 or Silquo Pro? What version? Which one is better?
